edsl.md

```diff
@@ -186,14 +186,6 @@ where `F1 : F2 : F3 : F4` is the (two's complement) byte-array representation of
       [concrete]
 ```
 
-``` {.k .symbolic .ceil}
-    rule #Ceil(#buf(@SIZE, @DATA))
-      =>      {(0 <=Int @SIZE) andBool #rangeBytes(@SIZE, @DATA) #Equals true}
-         #And #Ceil(@SIZE)
-         #And #Ceil(@DATA)
-      [anywhere, simplification]
-```
-
```

Comment:
Why is this not needed anymore?

Reply:
I added it to the description of the pull request.